configure.ac

```diff
@@ -203,6 +203,12 @@ AC_ARG_WITH([daemon],
   [build_bitcoind=$withval],
   [build_bitcoind=yes])
 
+AC_ARG_WITH([proving-service],
+  [AS_HELP_STRING([--with-proving-service],
+  [build zcash-proving-service (default is no)])],
```

Comment:
Non-blocking: Just spotted this should be `default=no` :stuck_out_tongue_closed_eyes: